File: 2310817120009_AlysaArmelia_PemrogramanI_Modul5/Bahasa C/PRAK505-2310817120009-AlysaArmelia.c
#include <stdio.h>

void Biodata (int tahunLahir, char Namaku[50], char Asal[99]) {
    
    int tahun_sekarang = 2023;
    int umur_seseorang = tahun_sekarang - tahunLahir;

    printf("\nPerkenalkan Nama Saya: %s\n", Namaku);
    printf("Umur Saya: %d\n", umur_seseorang);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);  
    printf("Asal Saya dari: %s\n", Asal);
}

int main() {
    int tahunLahir;
    char Namaku[50], Asal[99];
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&Namaku);
    scanf(" %[^\n]%*c",&Asal);
    Biodata(tahunLahir, Namaku, Asal);
return 0;
}

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul5/Python/PRAK505-2310817120009-AlysaArmelia.py
def Biodata(tahunLahir, Namaku, Asal):

    tahun_sekarang = 2023
    umur_seseorang = tahun_sekarang - tahunLahir

    print(f"\nPerkenalkan Nama Saya: {Namaku}")
    print(f"Umur Saya: {umur_seseorang}")
    print(f"Saya Adalah Angkatan: {tahun_sekarang}")
    print(f"Asal Saya dari: {Asal}")


tahunLahir = int(input())
Namaku = input()
Asal = input()
Biodata(tahunLahir, Namaku, Asal)